Makefile

```
@@ -8,7 +8,9 @@ GIT_SHA_FETCH := $(shell git rev-parse HEAD)
 export GIT_SHA=$(GIT_SHA_FETCH)
 
 install-dependencies: .install-logos
-	sudo dnf -y install python3-flask python3-flask-cors python3-flask-talisman
+	sudo dnf -y install python3-flask python3-flask-cors python3-flask-talisman npm
+	npm install -g corepack
+
```

Note sure which version but we can install NodeJS>16 to get corepack
```suggestion
	sudo dnf -y install python3-flask python3-flask-cors python3-flask-talisman nodejs

```

---

```
mfocko@ampere 
~ % node --version
v20.17.0
mfocko@ampere 
~ % corepack
zsh: corepack: command not found...
```

uhh…

---

Huh..

We can change it around to just install pnpm. `dnf install pnpm`. Might be better

---

Try this
```suggestion
	sudo dnf -y install python3-flask python3-flask-cors python3-flask-talisman pnpm
```
```
📦[spytec@fedora-rawhide]~% dnf search pnpm --showduplicates
Updating and loading repositories:
Repositories loaded.
Matched fields: name (exact)
 pnpm-0:9.11.0-1.fc42.noarch: Fast, disk space efficient package manager
```